### ISSUE TITLE
Unable to use alter_job_schedule without an enterprise license

### ISSUE THREAD

**Issue by od0** [timescale/timescaledb#1448]
**Relevant system information:**
 - OS: Ubuntu 16.04
 - PostgreSQL version: 11
 - TimescaleDB version: 1.4.1
 - Installation method: Docker `timescale/timescaledb-postgis:1.4.1-pg11`

**Describe the bug**
Ran into an issue where a continuous aggregate appeared to stop automatically refreshing. After debugging, it turned out that it's likely that a refresh job failed for whatever reason but with the retry_period set 12 hours by default, it continued to sit unrefreshed.

I ran the following query to reduce the retry_period to 10 minutes:

```sql
SELECT alter_job_schedule(job_id, schedule_interval => INTERVAL '10m', retry_period=> INTERVAL '10m')
FROM timescaledb_information.continuous_aggregate_stats 
WHERE view_name = 'much_cagg'::regclass;
```

This was the error that I got in response:

`ERROR:  cannot execute an enterprise function with an invalid enterprise license`

This seems like a core piece of functionality for relying on continuous aggregates and I'd love to be able to adjust this setting in non-enterprise installations. 

**To Reproduce** **Expected behavior** **Actual behavior**
Please see above :)

**Comment by gayyappan** [comment]
@od0 Did you try this statement to modify the interval?
ALTER VIEW device_summary SET (timescaledb.refresh_interval = '1 hour');
Using ALTER VIEW is recommended for modifying continuous aggregate related settings.

**Comment by cevian** [comment]
@od0 This is fixed in https://github.com/timescale/timescaledb/pull/1479 and will be available in the next release (1.5).
